RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -27,11 +29,22 @@ public class RileyLinkDeviceTableViewController: UITableViewController, TextFiel
         title = device.name
 
         self.observe()
+        
+        rssiFetchTimer = NSTimer.scheduledTimerWithTimeInterval(5, target: self, selector: #selector(RileyLinkDeviceTableViewController.updateRSSI), userInfo: nil, repeats: true)
```

Comment:
Minor – you can change `#selector(RileyLinkDeviceTableViewController.updateRSSI)` to just `#selector(updateRSSI)`